Lesson1_Ex3/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Lesson1_Ex3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Exercise 3.");
+            Console.WriteLine("Enter string 1");
+            var str1 = Console.ReadLine().ToUpper();
+            Console.WriteLine("Enter string 2");
+            var str2 = Console.ReadLine().ToUpper();
+            Console.WriteLine("Enter string 3");
+            var str3 = Console.ReadLine().ToUpper();
+
+            if (str1.StartsWith(str2))
```

Comment:
We could use string interpolation and pass boolean variable to reduce amount of code and unnecessary ifs